Kudu.Services.Web/Content/Scripts/FileBrowser.js

```diff
@@ -368,8 +360,7 @@ $.connection.hub.start().done(function () {
             viewModel.editText('');
             statusbar.fetchingContents();
             viewModel.fileEdit(this);
-            if(this.mime === "text/xml")
-            {
+            if (this.mime === "text/xml") {
                 Vfs.getContent(this)
                    .done(function (data) {
                        viewModel.editText(vkbeautify.xml(data));
```

Comment:
Do we still need to vkbeautify XML? I believe the Ace editor is already doing that if the extension is .xml (loads the XML mode). I'll test and confirm. vkbeautify was historically used here just to get a readable XML in the (old) editor, right?